src/Examples/UnitModels/Operations/heat_exchanger_0D_testing.ipynb

```diff
@@ -202,9 +253,67 @@
+      "2022-09-16 09:24:46 [INFO] idaes.init.fs.heat_exchanger: Initialization Completed, optimal - Optimal Solution Found\n",
+      "\n",
+      "====================================================================================\n",
+      "Unit : fs.heat_exchanger                                                   Time: 0.0\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Unit Performance\n",
+      "\n",
+      "    Variables: \n",
+      "\n",
+      "    Key            : Value      : Units                           : Fixed : Bounds\n",
+      "           HX Area :     50.000 :                      meter ** 2 :  True : (0, None)\n",
+      "    HX Coefficient :     500.00 : kilogram / kelvin / second ** 3 :  True : (0, None)\n",
+      "         Heat Duty : 1.2986e+06 :                            watt : False : (None, None)\n",
+      "\n",
+      "    Expressions: \n",
+      "\n",
+      "    Key             : Value  : Units\n",
+      "    Delta T Driving : 51.942 : kelvin\n",
+      "         Delta T In : 80.757 : kelvin\n",
+      "        Delta T Out : 23.128 : kelvin\n",
+      "\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                                      Units        shell Inlet shell Outlet tube Inlet tube Outlet\n",
+      "    Molar Flow (mol/s)               mole / second       100        100.00           -           -\n",
+      "    Mass Flow (kg/s)             kilogram / second    1.8015        1.8015           -           -\n",
+      "    T (K)                                   kelvin    450.00        373.13           -           -\n",
+      "    P (Pa)                                  pascal    101325    1.0132e+05           -           -\n",
+      "    Vapor Fraction                   dimensionless    1.0000       0.74886           -           -\n",
+      "    Molar Enthalpy (J/mol) Vap        joule / mole    50977.        48201.           -           -\n",
+      "    Molar Enthalpy (J/mol) Liq        joule / mole    13489.        7549.7           -           -\n",
+      "    Total Molar Flowrate             mole / second         -             -         250      250.00\n",
+      "    Total Mole Fraction benzene      dimensionless         -             -     0.40000     0.40000\n",
+      "    Total Mole Fraction toluene      dimensionless         -             -     0.60000     0.60000\n",
+      "    Temperature                             kelvin         -             -         350      369.24\n",
+      "    Pressure                                pascal         -             -  1.0132e+05  1.0132e+05\n",
+      "====================================================================================\n"
+     ]
+    }
+   ],
    "source": [
     "#Initialize the flowsheet, and set the output at WARNING\n",
```

Comment:
For the solver, it would be preferable to use the IDAES `get_solver` function for consistency.